flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -424,6 +435,9 @@ private String operatorName(String suffix) {
         case RANGE:
           LOG.warn("Fallback to use 'none' distribution mode, because {}={} is not supported in flink now",
```

Comment:
we need to move this log line into an else block

Reply:
this also reminds me that it might be good to add some info level logs to mention the distribution applied.

Reply:
OK, this warn has been move to else block and I add different info log to mention different distribution mode will distribute rows by which fields.